src/types/register.ts

```diff
@@ -38,7 +39,7 @@ export type RegisteredRouterRoute = RegisteredRouter['route']
  */
 export type RegisteredRoutes = Register extends { router: Router<infer TRoutes extends Routes> }
   ? TRoutes
-  : Route<string, '', Path<'', {}>, Query<'', {}>, false>[]
+  : Route<string, Host, Path<'', {}>, Query<'', {}>, false>[]
```

Comment:
If we don't need the empty string for host why do we need it for Path and Query?

```suggestion
  : Route<string, Host<''>, Path<'', {}>, Query<'', {}>, false>[]
```

Reply:
I don't think we needed it on Path and Query